desktop/cmp/form/DateInput.js

```diff
@@ -94,7 +115,7 @@ export class DateInput extends HoistInput {
     }
 
     parseDate(dateString) {
-        return moment(dateString, 'YYYY-MM-DD', true).toDate();
+        return moment(dateString, 'YYYY-MM-DD HH:mm:ss').toDate();
```

Comment:
Briefly discussed with Lee about making this format string a prop. Such that the app developer could enable timePicking and also specify how the user might be expected to manually enter values. The problem was that in the implementation we could set the formatter string to this time specific string but the datePicker popover would still pass a dateString of the form 'YYYY-MM-DD' and because we were passing the 'strict' param to moment it would throw.
Thinking maybe we still want to add a formatter string prop (to for example enable millis) but I also think removing the stict param makes sense. Not sure why we wouldn't want to except any user input that might be acceptable to moment.

Reply:
Actually should definitely still be a prop.